src/transactions/TransactionFrame.cpp

```diff
@@ -1208,6 +1208,11 @@ TransactionFrame::applyOperations(SignatureChecker& signatureChecker,
             }
         }
 
+        if (success && isSoroban())
+        {
+            success = applyLifetimeBumps(app, ltx);
```

Comment:
Can we do any pre-apply validation, like verify if bumps can be covered by the fees (maybe roughly?)